azure/Dockerfile

```diff
@@ -4,8 +4,8 @@
 FROM node:19-alpine as flyway
 
 ENV LANG C.UTF-8
-ENV JAVA_VERSION 21.0.1
-ENV JAVA_ALPINE_VERSION 21.0.1_p12-r0
+ENV JAVA_VERSION 17.0.6
+ENV JAVA_ALPINE_VERSION 17.0.6_p10-r0
```

Comment:
Should the java version be 17.0.8.1?

Reply:
It wasn't 17.0.8.1 before. We'll update this in a new PR

Reply:
yeah tbf I think it was me who forgot to update the docker images when increasing the java version in the main repo